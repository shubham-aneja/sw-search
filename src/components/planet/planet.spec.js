import React from 'react';
import { mount } from 'enzyme';
import Planet from './index.js'

describe('Planet', () => {
    it('rendering', () => {
        const mockedProps = {
            name: 'jimmy jonny',
            population: 1500000
        }
        const app = mount(<Planet {...mockedProps}/>)
        expect(app.html()).toMatchSnapshot();

    });

});

