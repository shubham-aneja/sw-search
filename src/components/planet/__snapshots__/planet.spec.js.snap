// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Planet rendering 1`] = `"<div class=\\"planet__container population-level-1\\"><div class=\\"planet__name\\">jimmy jonny </div><div class=\\"planet__population\\">Having population:<span class=\\"planet__population-count\\">1500000</span></div></div>"`;
